src/Models/Partner.ts

```diff
@@ -0,0 +1,26 @@
+import Xuid, { SupportedXuid } from '../Utils/Xuid';
+
+/**
+ * Business
+ */
+export default interface Partner extends Object {
+    _id: Xuid<SupportedXuid.Partner>;
+    business_name: string;
+    address_1?: string;
+    address_2?: string;
+    address_3?: string;
+    address_4?: string;
+    post_code?: string;
+    vat_code?: string;
+    invoice_name?: string;
+    country_code: string;
+    contact_name?: string;
+    contact_email?: string;
+    contact_no?: number;
+    website?: string;
+    logo?: string;
+    requires_gln?: boolean;
+    slug?: string;
+    external?: boolean;
+    public?: boolean;
+}
```

Comment:
Interfaces can extend each other... this is basically a business entity so
```typescript
export default interface Partner extends Business {
    _id: Xuid<SupportedXuid.Partner>
}
```

Reply:
You know, i tried to do this but couldnt work it out 😂 ta will put in now